### ISSUE TITLE
Issues with Data Being Saved By New Page In Mongodb

### ISSUE THREAD

**Issue by eganbailey** [jedireza/drywall#400]
Hi,

I tried making a new page off of the account and settings pages. I've added a schema, and I feel like I've mirrored the routes, both index.js files (in views/account and public/views/account), and l linked my new Schema through models.js .

However, even though I have a new collection in Mongodb, it remains empty, and the input to the new page remains unsaved no matter what I do. Do you know what I'm doing wrong? I could show you the files in question if you'd like. 

Basically, imagine you're re-making the settings page, but instead of trying to put data in the "accounts" collection, you're putting it in a different collection.

Any help would be much appreciated,

Egan

**Comment by jedireza** [comment]
Thanks for opening an issue and pardon the late reply.

> Do you know what I'm doing wrong? I could show you the files in question if you'd like.

It sounds like you're doing the right steps. Have you gotten past this since creating the issue?

**Comment by jedireza** [comment]
Drywall has been moved to Aqua. Closing this issue.
